examples/testimonials/readme.md

```diff
@@ -0,0 +1,15 @@
+# Forms example - Testimonial
+This is an example of using a form submission to automatically create a piece type in your Apostrophe 3 project.
```

Comment:
I have to admit that even though this is a very clear sentence, it took me a couple of read-throughs to really understand what this example did. Do you think you can expand this introduction just a little? Something along the lines of, "The form module can be an easy way to collect information from front-end users. This example demonstrates how to take that information and use it to automatically populate the fields of an existing piece type in your Apostrophe 3 project."